docs/guide/acquiring_data/helioviewer.rst

```diff
@@ -23,59 +23,97 @@ the API using the same parameters one would use when making a web request.
 Nearly all requests require the user to specify the data they are interested in
 and this can be done using one of two methods:
 
-1. Call "get_data_sources()" to get a list of the data that is available, and use the source id numbers referenced in the result to refer to a particular dataset, or,
+1. Call "get_data_sources()" to get a list of the data that is available and using "get_source_id()" to get the source id numbers which
```

Comment:
Is this still true? Do we need get_data_sources?

Reply:
Shall we remove this because now we only use the four parameters to request the API?